src/test/java/pages/CartPage.java

```diff
@@ -0,0 +1,81 @@
+    private static final By PRICE_OF_THIRD = By.cssSelector("tbody>tr:nth-child(3)>td[class=\"cart_total\"]>span");
+    private static final By PRICE_OF_SHIPPING = By.cssSelector("#total_shipping");
+
+    private final WebDriver driver = WebDriverSingletion.getInstance().openBrowser();
+
+    public CartPage addToCart() {
+        driver.findElement(PRODUCT_DRESS_CHOICE).click();
+        driver.findElement(ADD_TO_CART_BUTTON).click();
+        driver.findElement(CONTINUE_SHOPPING_BUTTON).click();
+        driver.findElement(OPEN_DRESSES_BUTTON).click();
+        driver.findElement(PRODUCT_DRESS3_CHOICE).click();
+        driver.findElement(ADD_TO_CART_BUTTON).click();
+        driver.findElement(CONTINUE_SHOPPING_BUTTON).click();
+        driver.findElement(OPEN_DRESSES_BUTTON).click();
+        driver.findElement(PRODUCT_DRESS2_CHOICE).click();
+        driver.findElement(ADD_TO_CART_BUTTON).click();
+        driver.findElement(PROCEED_TO_CHECKOUT_BUTTON).click();
+        return new CartPage();
+    }
+
+    public double getTotalPrise() {
+        String priceTotal = driver.findElement(TOTAL_PRICE).getText();
+        String[] split = priceTotal.split("[\\$]");
+        String delimiter = "";
+        String totalString = String.join(delimiter, split);
+        double totalPrice = Double.parseDouble(totalString);
+        return totalPrice;
+    }
+
+    public double getFirstPrise() {
+        String priceFirst = driver.findElement(PRICE_OF_FIRST).getText();
+        String[] splitFirst = priceFirst.split("[\\$]");
+        String delimiterFirst = "";
+        String stringFirst = String.join(delimiterFirst, splitFirst);
+        double priseFirst = Double.parseDouble(stringFirst);
+        return priseFirst;
+    }
+
+    public double getSecondPrise() {
+        String priceSecond = driver.findElement(PRICE_OF_SECOND).getText();
+        String[] splitSecond = priceSecond.split("[\\$]");
+        String delimiterSecond = "";
+        String stringSecond = String.join(delimiterSecond, splitSecond);
+        double priseSecond = Double.parseDouble(stringSecond);
+        return priseSecond;
+    }
+
+    public double getThirdPrise() {
+        String priceThird = driver.findElement(PRICE_OF_THIRD).getText();
+        String[] splitThird = priceThird.split("[\\$]");
+        String delimiterThird = "";
+        String stringThird = String.join(delimiterThird, splitThird);
+        double priseThird = Double.parseDouble(stringThird);
+        return priseThird;
+    }
+
+    public double getShippingPrise() {
+        String priceShip = driver.findElement(PRICE_OF_SHIPPING).getText();
+        String[] splitShip = priceShip.split("[\\$]");
+        String delimiterShip = "";
+        String stringShip = String.join(delimiterShip, splitShip);
+        double priseShip = Double.parseDouble(stringShip);
+        return priseShip;
+    }
```

Comment:
This code is just duplication. Create private method, which gets By as parameter and all other methods will look like:
`public double getShippingPrice() {
return getPrice(PRICE_OF_SHIPPING);
}`